plans/placebo/go.mod

```diff
@@ -1,5 +1,5 @@
-module github.com/ipfs/testground/plans/placebo
+module github.com/testground/testground/plans/placebo
 
 go 1.14
 
-require github.com/ipfs/testground/sdk/runtime v0.4.0
+require github.com/testground/testground/sdk/runtime v0.4.0
```

Comment:
Actually this will break :-( There is no such module. We will have to spin off the sdk into `testground/sdk-go` and make a first release, to then fix this require.